src/apps/AppCenter/InstalledApps/AppContent.js

```diff
@@ -2,224 +2,265 @@ import React from 'react'
-  const { below, breakpoints } = useViewport()
-  const compact = appBelow(below, breakpoints.medium)
+      repoAddress,
+      repoName,
+    } = repo
+    const repoDetails = useRepoDetails(baseUrl, detailsUrl)
+    const canUpgrade = currentVersion.version !== latestVersion
+    const { below, breakpoints } = useViewport()
+    const compact = appBelow(below, breakpoints.medium)
 
-  return (
-    <div
-      css={`
-        padding: ${6 * GU}px ${4 * GU}px ${8 * GU}px;
-      `}
-    >
-      <div
-        css={`
-          display: flex;
-          justify-content: space-between;
-          margin-bottom: ${6 * GU}px;
-          overflow: hidden;
-          flex-direction: ${compact ? 'column' : 'row'};
-          align-items: ${compact ? 'flex-start' : 'flex-end'};
-        `}
-      >
-        <div
-          css={`
-            display: flex;
-            align-items: center;
-          `}
-        >
-          <div
-            css={`
-              margin: 0 ${3 * GU}px 0 0;
-            `}
-          >
-            <AppIcon app={{ baseUrl, icons }} size={80} />
-          </div>
-          <div>
-            <h1
-              css={`
-                white-space: nowrap;
-                margin-bottom: ${1 * GU}px;
-                font-size: 22px;
-              `}
-            >
-              {name}
-            </h1>
+    return (
+      <React.Fragment>
+        <Bar>
+          <BackButton onClick={onClose} />
+        </Bar>
+        <Split
+          primary={
+            <Box>
+              <div
+                css={`
+                  display: grid;
+                  grid-template-columns: auto;
+                  grid-gap: ${4 * GU}px;
+                `}
+              >
+                <div
+                  css={`
+                    display: grid;
+                    grid-template-columns: auto auto 1fr;
+                    grid-gap: ${2 * GU}px;
 
-            {author && (
-              <React.Fragment>
-                <Heading2>Created by</Heading2>
-                <div>
-                  <LocalIdentityBadge entity={author} />
+                    display: flex;
+                    flex-direction: row;
+                    flex-wrap: wrap;
+                    align-items: center;
+                    justify-content: space-between;
+                  `}
+                >
+                  <div
+                    css={`
+                      display: grid;
+                      grid-template-columns: auto auto;
+                      grid-gap: ${2 * GU}px;
+                    `}
+                  >
+                    <AppIcon app={{ baseUrl, icons }} size={80} radius={12} />
+                    <div
+                      css={`
+                        display: flex;
+                        flex-direction: column;
+                        align-items: flex-start;
+                        justify-content: center;
+                        margin-right: ${2.5 * GU}px;
```

Comment:
Do we need the margin-right here (since this is already the last item in the row)?

Reply:
Not the last when the `Upgrade button` appears:
![image](https://user-images.githubusercontent.com/3072458/63752949-6ffb8800-c8b2-11e9-9066-f21b8aebbca8.png)
This helps keep some distance with that button until it wraps.